hazelcast-hibernate53/src/main/java/com/hazelcast/hibernate/local/TimestampsRegionCache.java

```diff
@@ -80,15 +83,19 @@ protected void maybeInvalidate(final Object messageObject) {
             final Expirable value = cache.get(key);
             final Long current = value != null ? (Long) value.getValue() : null;
             if (current != null) {
-                if (ts.getTimestamp() > current) {
-                    if (cache.replace(key, value, new Value(value.getVersion(), nextTimestamp(), ts.getTimestamp()))) {
+                if (ts.getTimestamp() > current ) {
+                    logger.fine(String.format("Invalidating entry for key %s, ", ts.getKey()));
```

Comment:
We better make it finest and wrap with `if (log.isFinestEnabled())`

Reply:
Sure